config/google2fa.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+return [
+
+    /*
+     * Enable / disable Google2FA.
+     */
+    'enabled' => true,
+
+    /*
+     * Lifetime in minutes.
+     *
+     * In case you need your users to be asked for a new one time passwords from time to time.
+     */
+    'lifetime' => 0, // 0 = eternal
```

Comment:
Is there an upside to having only limited lifespan? If so, you could set `lifetime` to something big (1 month) and still let `keep_alive` true.

But I am not sure what best practices are concerning the lifetime of 2FA.

Reply:
not sure if this is a good choise to change, i noticed that the google authenticator does not respect all possible options. This can cause invalid codes.